yamux/src/frame/io.rs

```diff
@@ -20,24 +20,24 @@ use std::{
     task::{Context, Poll},
 };
 
+const MAX_FRAME_BODY_LEN: usize = 1024 * 1024;
```

Comment:
Perhaps add a comment explaining this constant.

Reply:
:+1: cf76ccf

Note that this pull request does not change the value, but instead makes the previous default value constant. Previously this was set through `max_receive_buffer`. With https://github.com/libp2p/rust-yamux/issues/175 this configuration flag is no longer needed.